__std__/cells/plutus-apps/packages/combined-plutus-apps-haddock.nix

```diff
@@ -24,11 +24,7 @@ cell.library.combine-haddock {
 
       == Handy module entrypoints
 
-        * "PlutusTx": Compiling Haskell to PLC (Plutus Core; on-chain code).
```

Comment:
These just don't want to work. 

This `prologue` string is passed to `__std__/cells/plutus-apps/library/combined-haddock.nix:86`:
```
  # Generate the contents and index
  ${ghc}/bin/haddock \
    --gen-contents \
    --gen-index \
    --quickjump \
    ${lib.optionalString (prologue != null) "--prologue ${prologue}"} \
    "''${interfaceOpts[@]}"
```

`ghc` will parse it and replace stuff in double quotes with links, but it seems to work only for modules within the current project.
For example it works for `Plutus.Contract` (coming from `plutus-apps`) but not for `PlutusTx.Prelude` (coming from `plutus`).

However, links to `plutus` packages (including `plutus-tx`) work internally  within haddock coming from `plutus-apps` modules.

Reply:
Yes, these links are from before the plutus/plutus-apps split. It's fine to remove them.